src/types.rs

```diff
@@ -200,8 +197,9 @@ mod tests {
     fn test_deserialise_no_time() {
         let s = r#"{"access_token":"abc123"}"#;
         let token: Token = serde_json::from_str(s).unwrap();
+        let expires = OffsetDateTime::now_utc() + Duration::seconds(3600);
```

Comment:
I'm curious how/where we actually depend on deserializing an `InnerToken` without time? Seems like deserialization would always have an `expires_at`?

Reply:
Wow, good catch. I totally misread this test case. I'll have a look at the different auth methods to confirm, but I suspect you're right that deserialization will always have `expires_at`. If it does, I'll just update this case to cover  `Token::from_string`. Otherwise I'll add a new case to cover it.